src/map/entities/charentity.h

```diff
@@ -34,6 +34,7 @@ This file is part of DarkStar-server source code.
 
 #include "battleentity.h"
 #include "petentity.h"
+#include "charentity.h"
```

Comment:
charentity.h includes charentity.h?

Reply:
and trustentity.h I think can be forward declared in here still as well

Reply:
Fixed in https://github.com/DarkstarProject/darkstar/pull/4729/commits/13197f80df3b1eb5f9e0027eb07d9a5358ddc690

`trustentity.h` can't be forward declared without everything breaking.

Reply:
Fixed in https://github.com/DarkstarProject/darkstar/pull/4729/commits/948344cef06c3569742b020b432e96d18d8280dc